src/main/kotlin/no/nav/familie/ba/soknad/api/clients/mottak/MottakClient.kt

```diff
@@ -37,25 +36,7 @@ class MottakClient(
     }
 
     fun sendSøknad(søknad: Søknad): Ressurs<Kvittering> {
-        val uri: URI = URI.create("$mottakBaseUrl/api/soknad/v4")
-
-        try {
-            val multipartBuilder = MultipartBuilder().withJson("søknad", søknad)
-
-            val response = postForEntity<Ressurs<Kvittering>>(
-                uri = uri,
-                payload = multipartBuilder.build(),
-                httpHeaders = MultipartBuilder.MULTIPART_HEADERS
-            )
-            LOG.info("Sende søknad til mottak OK: ${response.data}")
-            return response
-        } catch (e: Exception) {
-            throw IllegalStateException("Sende søknad til familie-ba-mottak feilet", e)
-        }
-    }
-
-    fun sendSøknadV3(søknad: SøknadV3): Ressurs<Kvittering> {
-        val uri: URI = URI.create("$mottakBaseUrl/api/soknad/v3")
+        val uri: URI = URI.create("$mottakBaseUrl/api/soknad/v5")
```

Comment:
Hvorfor trengs ikke dette lengre?

Reply:
Fordi deploy-planen belager seg på å skru på killswitcher og deploye alt før vi skrur på igjen, og at den nye modell-funksjonaliteten i frontend forhindrer gamle søknader fra å komme hit, derfor trenger vi bare å forwarde søknader på siste kontraktformat til mottak. Controlleren for å ta imot v4-søknader returenerer bare 410 gone nå, og v3-controlleren fjernet vi for lenge siden.